app/javascript/mastodon/features/ui/components/column.js

```diff
@@ -48,7 +49,7 @@ class Column extends React.PureComponent {
     this._interruptScrollAnimation = scrollTop(scrollable);
   }
 
-  handleWheel = () => {
+  handleScroll = () => {
```

Comment:
I think that a shape like `handleScroll = debounce(() => { /* ... */ })` is better.